minidump-processor/src/process_state.rs

```diff
@@ -416,6 +421,48 @@ impl PossibleBitFlip {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum CrashReasonInconsistency {
+    AccessViolationWhenAccessAllowed,
+    CrashAddressNotFoundInMemoryAccesses,
+    // INCOMPLETE: Struct fields are for human output
+    // might be a bit redundant if memory_operation is printed anyways?
+    InconsistentMemoryOperation {
+        crash_reason_operation: MemoryOperation,
+        instruction_operation: MemoryOperation,
+    },
+}
+
+impl std::fmt::Display for CrashReasonInconsistency {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            CrashReasonInconsistency::AccessViolationWhenAccessAllowed => {
+                write!(
+                    f,
+                    "Crash reason is access violation exception but access is allowed"
+                )
+            }
+            CrashReasonInconsistency::CrashAddressNotFoundInMemoryAccesses => {
+                write!(
+                    f,
+                    "Crash reason address not found in memory accesses done by crashing instruction"
+                )
+            }
+            CrashReasonInconsistency::InconsistentMemoryOperation {
+                crash_reason_operation,
+                instruction_operation,
+            } => {
+                write!(
+                    f,
+                    "Crash reason is {} but instruction is {}",
+                    crash_reason_operation.to_string(),
```

Comment:
The `to_string` calls here are redundant since these types impl `Display` (clippy will warn you about this depending on the configuration).